modules/seqexec/server/src/main/resources/Tcs.xml

```diff
@@ -1821,6 +1821,11 @@
             <description>AO fold position</description>
             <top>ag</top>
         </attribute>
+        <!--<attribute name="useAo">-->
+            <!--<channel>im:AOConfigFlag.VAL</channel>-->
+            <!--<type>STRING</type>-->
+            <!--<description>Using AO flag</description>-->
+        <!--</attribute>-->
```

Comment:
Added the attribute only to find that it was an Enum (which are not supported in the xml configuration file). I left it commented to keep the information about the channel.